app/helpers/playlists_helper.rb

```diff
@@ -17,22 +17,12 @@ def playlist_cover_div
     content_tag(:div, '', class: 'no_pic')
   end
 
-  # Returns true when the Pic with this ID does not have a greenfield variant.
-  def no_greenfield_variant?(pic_id)
-    (69806..72848).cover?(pic_id)
-  end
-
-  # Returns true when the Pic with this ID does not have a greenfield nor an original variant.
-  def no_greenfield_and_original_variant?(pic_id)
-    pic_id < 69807
-  end
```

Comment:
@Manfred I'm seeing problems in production related to this (not sure if this is cause) — for example check out all of this guy's playlists: https://alonetone.com/thetworegs — we don't seem to be scaling up the smaller image anymore.

Reply:
Yes, I noticed it too with my own playlists. I'm not sure what is wrong here unless I can browse around in the production database. Maybe we can screen share later today and you can poke around a bit?